binary_search_tree/BinarySearchTree.py

```diff
@@ -0,0 +1,179 @@
+# Node for Binary Search Tree
+class Node:
+
+    def __init__(self, data):
+        self.data = data
+        self.left = None
+        self.right = None
+
+# function for find the left side right most node in binary search tree
```

Comment:
The code does not comply to PEP8.

*Origin: PEP8Bear, Section: `pep8`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpc9jm3hgg/binary_search_tree/BinarySearchTree.py
+++ b/tmp/tmpc9jm3hgg/binary_search_tree/BinarySearchTree.py
@@ -7,6 +7,8 @@
         self.right = None
 
 # function for find the left side right most node in binary search tree
+
+
 def find_leftside_rightmost(node):
     left_side_rightmost = node.left
     prev_node = node
```